scripts/flush.py

```diff
@@ -0,0 +1,119 @@
+import requests
+import json
+import time
+import sys
+import os
+
+sys.path.append(os.getcwd())
+
+from app import config
+from uuid import uuid4
+from app.eq import build_response_id
+from sdc.crypto.encrypter import encrypt
+from app import jwt
+
+collection_ex = sys.argv[1:]
+print("Collection exercise: " + str(collection_ex))
+
+
+if not os.getenv('APP_SETTINGS'):
+    config_info = config.DevelopmentConfig
+else:
+    config_info = config.BaseConfig
+
+# Put config into a dict
+config = dict((name, getattr(config_info, name)) for name in dir(config_info) if not name.startswith('__'))
+
+# Setup URLs
+case_url = f"{config['CASE_URL']}/cases/"
+ci_url = f"{config['COLLECTION_INSTRUMENT_URL']}/collection-instrument-api/1.0.2/collectioninstrument/id/"
+collex_url = f"{config['COLLECTION_EXERCISE_URL']}/collectionexercises/"
+sample_url = f"{config['SAMPLE_URL']}/samples/"
+eq_url = f"{config['EQ_URL']}"
+
+collection_ex_info = requests.get(collex_url + "link/" + collection_ex[0],
```

Comment:
Anything IO would be do well to sit in a main function so that if this script is ever imported, the requests aren't performed at import-time, for example.